### PR TITLE
Fix Italian stations

### PR DESCRIPTION
**Pull request by lucavezz**
Disable a lot of closed Italian train stations, plus small fixes in typo and is_main_station settings.

### CODE REVIEW

File: stations.csv [hunk 1]
```diff
@@ -16894,7 +16894,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 18834;Silandro/Schlanders;silandro-schlanders;8302205;;46.6259230000;10.7664880000;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;8300483;t;;f;;f;;f;;f;;f;8302205;t;S02205;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 18835;Tirano;tirano;8306719;;46.2093000000;10.1666820000;18820;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;8302436;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 18836;Venezia Mestre (Viale Stazione);venezia-mestre-viale-stazione;;;45.4821090000;12.2346420000;;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;8324253;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
-18837;Malles / Mals;malles-mals;8362035;;46.6866100000;10.5421530000;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;8362035;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+18837;Malles / Mals;malles-mals;8362035;;46.6866100000;10.5421530000;;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;8362035;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```
Comment (avandecreme) [inline stations.csv]: Maybe set trenitalia_enabled to false as well?

File: stations.csv [hunk 2]
```diff
@@ -27190,7 +27190,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 29337;Agrigento;agrigento;;;37.31138095340275;13.58164131467822;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;;f;ITAGR;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 29338;Bolzano/Bozen;bolzano-bozen;;;46.4961167;11.3546747;;IT;Europe/Rome;t;f;f;f;f;f;;;f;;f;;f;;f;ITBZO;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 29339;Messina;messina;;;38.18963375;15.55735599999997;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;;f;ITMES;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
-29340;Cerignola;cerignola;;;41.26185045;15.907984650000005;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITCER;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+29340;Cerignola;cerignola;;;41.26185045;15.907984650000005;;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;ITCER;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```
Comment (avandecreme) [inline stations.csv]: It looks like this one is used for Distribusion, it should not get disabled.

File: stations.csv [hunk 3]
```diff
@@ -27099,7 +27099,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 29245;Andria;andria;;;41.2242;16.2969;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;;f;ITAND;t;9358;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 29246;Treviso;treviso;;;45.6668893;12.2430437;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;;f;ITTRO;t;10478;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 29247;Cervia;cervia;;;44.2635492;12.3476822;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;;f;ITCEV;t;12968;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
-29248;Porto Garibaldi;porto-garibaldi;;;44.678742;12.241495;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;;f;ITPGA;t;9668;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+29248;Porto Garibaldi;porto-garibaldi;;;44.678742;12.241495;;IT;Europe/Rome;t;f;f;f;f;f;;;f;;f;;f;;f;ITPGA;t;9668;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```
Comment (avandecreme) [inline stations.csv]: Distribusion too

File: stations.csv (hunk 2 above)
Comment (lucavezz) [reply stations.csv]: Oops, I'll re-enable the station.

File: stations.csv (hunk 3 above)
Comment (lucavezz) [reply stations.csv]: I'll re-enable this one too.

File: stations.csv (hunk 1 above)
Comment (lucavezz) [reply stations.csv]: Uhm, why exactly?

File: stations.csv [hunk 4]
```diff
@@ -19081,7 +19081,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 21056;Biella-Chiavazza;biella-chiavazza;8300970;;45.5680974;8.0668666;;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;8300970;t;S00970;;;f;;f;;f;;f;ITAWI;f;f;;;;;;;;;;;;;;;;;;;
 21057;Binetto;binetto;8354003;;41.0238906;16.7031294;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;8354003;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 21058;Catanzaro Città;catanzaro-citta;8354228;;38.9132917;16.5844075;29342;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;8354228;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;カタンツァーロ;;;;Катандзаро;;;
-21059;Catanzaro Pratica;catanzaro-pratica;8354300;;38.9090716;16.5871781;29342;IT;Europe/Rome;f;t;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;8354300;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+21059;Catanzaro Pratica;catanzaro-pratica;8354300;;38.9090716;16.5871781;29342;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;8354300;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```
[the remark this answers is not in the document]
Comment (avandecreme) [reply stations.csv]: nevermind, bad eyes.

**Pull request merged**